frontend/public/components/catalog.jsx

```diff
@@ -55,7 +56,7 @@ class CatalogListPage extends React.Component {
       const tileIconClass = tileImgUrl ? null : iconClass;
       const tileDescription = _.get(serviceClass, 'spec.description');
       const tileProvider = _.get(serviceClass, 'spec.externalMetadata.providerDisplayName');
-      const href = `/k8s/cluster/clusterserviceclasses/${serviceClass.metadata.name}/create-instance`;
+      const href = `/k8s/cluster/clusterserviceclasses/${serviceClass.metadata.name}/create-instance?preselect-ns=${namespace}`;
```

Comment:
When navigating to the Deploy Image page, we use the query string parameter `ns`.  Wondering if we should aim to be consistent with the parameter name?  Nit:  the grammarian finds "preselect-ns" a bit awkward.   "preselected-ns" is better?